proto/greptime/v1/meta/common.proto

```diff
@@ -39,6 +39,7 @@ message TableName {
   string catalog_name = 1;
   string schema_name = 2;
   string table_name = 3;
+  uint64 table_id= 4;
```

Comment:
Why define `table_id` here since it's already in `RegionStat`?

Reply:
Got it.